nautobot/core/graphql/schema.py

```diff
@@ -211,6 +211,12 @@ def extend_schema_type_custom_field(schema_type, model):
     if settings.GRAPHQL_CUSTOM_FIELD_PREFIX and isinstance(settings.GRAPHQL_CUSTOM_FIELD_PREFIX, str):
         prefix = f"{settings.GRAPHQL_CUSTOM_FIELD_PREFIX}_"
 
+    # clear old custom fields; TODO: safe?
```

Comment:
Only concern here is that if we don't have a prefix defined (though IMHO we should remove that as a possibility and require it to always be defined, as otherwise we're almost guaranteed collisions), the below will remove *all* resolvers. Same comment on the other two cases below.